wrappers/dotnet/Directory.Build.props

```diff
@@ -0,0 +1,27 @@
+<Project>
+    <!-- Set common properties regarding assembly information and nuget packages -->
+    <PropertyGroup>
+        <Authors>Tomislav Markovski</Authors>
+        <Company>Hyperledger</Company>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/mattrglobal/ffi-bbs-signatures</PackageProjectUrl>
+        <PackageTags>BBS+ Ursa ZKP</PackageTags>
+        <Product>BBS+ Signatures</Product>
+        <RepositoryUrl>https://github.com/mattrglobal/ffi-bbs-signatures.git</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <Version>1.0.0</Version>
```

Comment:
How do feel about keeping the version pre-1.0 i.e if we decide to keep the wrappers minor version in lockstep with each other then the first released version would be 0.1.0?